packages/did-auth-siop-op-authenticator/README.md

```diff
@@ -115,7 +115,7 @@ const opSession = await agent.removeSessionForSiop({
 })
 ```
 
-### Authenticate with DID auth SIOP:
+### Authorize with DID auth SIOP:
```

Comment:
Although these are called authorization requests. The SIOP key binding results in actual authentication, so do not simply change the word everywhere

Reply:
reverted